bluetooth/src/commonMain/kotlin/Bluetooth.kt

```diff
@@ -51,6 +51,10 @@ import kotlinx.coroutines.flow.transformLatest
 import kotlin.coroutines.CoroutineContext
 import kotlin.jvm.JvmName
 
+private val defaultBluetoothDispatcher by lazy {
```

Comment:
Not a fan of global variable, can be nested in the interface itself?

Reply:
the reason it's there is that the `1.6.0-native-mt` version of the coroutines does not support all the features from `1.6.0` when migrated to the next version the global field shall be removed. so I'd say it's intentionally ugly :-)

Reply:
I'd actually argue a private top level property is fine, but [should this not be `@SharedImmutable`](https://kotlinlang.org/docs/multiplatform-mobile-concurrency-overview.html#global-properties), what happens when you access this on a background thread on iOS @vpodlesnyak ?

Reply:
agree. updating..